modelling/src/main/java/org/axonframework/modelling/command/inspection/ForwardingMessageHandlingMember.java

```diff
@@ -0,0 +1,21 @@
+package org.axonframework.modelling.command.inspection;
+
+import org.axonframework.commandhandling.CommandMessageHandlingMember;
+import org.axonframework.messaging.Message;
+
+/**
+ * Interface describing a message handler capable of forwarding a specific command.
+ *
+ * @since 4.6
```

Comment:
Two pointers here.
The new release this change would become a part of is 4.6.0.
Secondly, you've added this new class. Hence, the `@author` tag carrying your name (and honor) is in place. :-)